auth/README.md

```diff
@@ -1,19 +1,31 @@
 # auth
 
-This directory contains scripts that can be used to set environment variables
-containing credentials and server information. These environment variables are
-used by `registry` and other tools to configure their connections to the
+This directory contains scripts that can be used to create registry configurations
+containing credentials and server information for connections to the
 `registry-server`.
 
-- [LOCAL.sh](LOCAL.sh) configures clients to work with a locally-running
-  `registry-server`.
-- [CLOUDRUN.sh](CLOUDRUN.sh) configures clients to work with a
-  `registry-server` deployed to Cloud Run using 'make build' and 'make deploy'
-  from the top-level [Makefile](../Makefile).
-- [ENVOY.sh](ENVOY.sh) configures clients to work with a locally-running
-  `registry-server` that is proxied behind a local Envoy instance.
-- [GKE.sh](GKE.sh) configures clients to work with a `registry-server` deployed
-  to GKE. For more details about GKE deployments, please refer to
-  [deployments/gke/README.md](../deployments/gke/README.md).
-- [HOSTED.sh](HOSTED.sh) configures clients to work with a `registry-server`
-  hosted on Google Cloud.
+- [LOCAL.sh](LOCAL.sh) creates a configuration ("local") that configures the registry
+  client to work with a locally-running server.
+- [HOSTED.sh](HOSTED.sh) creates a configuration ("hosted") that configures the registry
+  client to work with a Google Cloud hosted server.
+
+Each script should only be run once to create and activate a configuration. Once
+created, the configurations can be selected be using the registry commands. For
+example:
+
+$ registry config configurations activate local
```

Comment:
So... we're adding an extra step. That means that just using the scripts as-before won't work. WDYT about adding an activate step at the end of each script?

Reply:
No, they will work. When you create a new profile, it activates it.

Reply:
ok I see now "create and activate" - LGTM